css/_side_toolbar_container.scss

```diff
@@ -2,7 +2,7 @@
  * Toolbar side panel main container element.
  */
 #sideToolbarContainer {
-    background-color: rgba(0,0,0,0.8);
+    background-color: $sideBarBackground;
```

Comment:
The id says sideToolbar but the variable is sideBar. Should the naming be consistent? What is a side toolbar?